templates/frag/operation.hbs

```diff
@@ -27,10 +27,6 @@ public enum {{className name}}Result {
     {{/if}}
     case _{{{code}}}{{#if defaultContent.nativeType}}(_ value: {{{defaultContent.nativeType}}}){{/if}}
```

Comment:
Ooh! We need to omit `401` if it's in here, as we'll never respond with that status code.